features/opennms-es-rest/src/main/java/org/opennms/plugins/elasticsearch/rest/EventToIndex.java

```diff
@@ -270,6 +283,7 @@ private Index createEventIndexFromEvent(final Event event) {
 		// Parse event parameters
 		handleParameters(event, body);
 
+		body.put("EventForwarder", eventIndexName);
```

Comment:
I'm not sure what this is for, but perhaps it should only be added in the event the index name is not empty?